cdap-api/src/main/java/co/cask/cdap/api/schedule/Schedule.java

```diff
@@ -26,14 +26,47 @@
 
   private final String description;
 
+  // NOTE: the two below attributes are left for backwards compatibility
   private final String cronEntry;
 
+  /**
+   * Build a time-based schedule.
+   *
+   * @param name name of the schedule
+   * @param description description of the schedule
+   * @param cronExpression cron expression for the schedule
+   * @return a schedule based on the given {@code cronExpression}
+   */
+  public static Schedule buildTimeSchedule(String name, String description, String cronExpression) {
```

Comment:
Id prefer to have a TimeSchedule.of(...) than this. Is there an advantage to this way?